controller/src/main/java/org/jboss/as/controller/services/path/PathManagerService.java

```diff
@@ -265,18 +265,12 @@ final void removePathEntry(final String pathName, boolean check) throws Operatio
      * @param pathName the name of the relevant path. Cannot be {@code null}
      * @param path  the value of the path within the model. This is either an absolute path or
      *              the relative portion of the path. Cannot be {@code null}
-     * @param possiblyAbsolute {@code true} if the path may be absolute and a check should be performed before installing
-     *                         a service variant that depends on the service associated with {@code relativeTo}
      * @param relativeTo the name of the path this path is relative to. If {@code null} this is an absolute path
      * @return the service controller for the {@code Service<String>}
      */
     final ServiceController<?> addRelativePathService(final ServiceTarget serviceTarget, final String pathName, final String path,
-                                                      final boolean possiblyAbsolute, final String relativeTo) {
```

Comment:
This seems like a API breaking change.
The removal of the `possiblyAbsolute` changes the behaviour of the method. Without it, it is no longer possible for callers to get an AbsolutePathService from this method.